packages/core/test/render3/definition_spec.ts

```
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component, Directive, Pipe} from '@angular/core/public_api';
+
+import {isStandalone} from '../../src/render3/definition';
+
+describe('definition isStandalone()', () => {
+  it('should check is component is standalone', () => {
+    @Component({selector: 'standalone-cmp', standalone: true})
```

I think it'd be great to add some tests for a case when non-annotated class is used as an argument in the `isStandalone` function.

---

Actually, if a non-annotated class is used as an argument, we should probably throw an error, similar to this one (and have tests that verify this behavior):

https://github.com/angular/angular/blob/764fa3d9c37eb70acd21879296ec039de07173ea/packages/core/src/render3/errors.ts#L36-L39